components/Containers.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components/native'
+
+export const MainContainer = styled.View`
+  flex: 1;
+`
+export const TopContainer = styled.View`
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  background-color: whitesmoke;
+  padding-top: 70px;
+`
+export const BottomContainer = styled.View`
+  flex: 1;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: #d6bea0;
+  width: 100%;
+  padding-top: 20px;
+`
+export const ColorContainer1 = styled.View`
```

Comment:
That's not really descriptive name - next time try to find different indicator than number 1, 2, 3